src/modules/leaderboard-api.js

```diff
@@ -0,0 +1,33 @@
+class Leaderboard {
+  constructor(inputName, inputScore) {
+    this.inputName = inputName;
+    this.inputScore = inputScore;
+  }
+
+  createScores = async () => {
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/gM5SPU2u7B8UfGoR9LJt/scores/', {
+      method: 'POST',
+      body: JSON.stringify({
+        user: this.inputName.value,
+        score: this.inputScore.value,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    });
+    const data = await response.json();
+    return data;
+  };
+
+  getScores = async () => {
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/gM5SPU2u7B8UfGoR9LJt/scores/',
+      {
+        method: 'GET',
+      });
+    const data = await response.json();
+    return data;
+  };
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export { Leaderboard };
```

Comment:
- [ Required ] I think it would be better if you don't disable linters in your source files, disabling linters will lead to writing code that does not follow best JavaScript practices. I would like to suggest that you use a `default` export in this case instead of a `named` export. This can be done as shown below.

## leaderboard.js
```JavaScript
export default Leaderboard;
```
## add-to-list.js
```JavaScript
import Leaderboard from './leaderboard-api.js';
```